src/core/text/TextMetrics.js

```diff
@@ -0,0 +1,191 @@
+import Text from './Text';
```

Comment:
This feels... wrong, as there's now a circular dependency between Text and TextMetrics.

The only thing you are using from there are 2 static functions, getFontStyle and calculateFontProperties.
Either those could be moved here, or perhaps better, we could have another new file which just contains text utils? This could be where TextMetrics and measure live, and getFontStyle, calculateFontProperties and wordWrap could go in textUtils.js? @bigtimebuddy again?